### DISCUSSION TITLE
Elasticsearch not using custom ingest

### DISCUSSION THREAD

**Discussion by sturge33** [Security-Onion-Solutions/securityonion#3051] [Unsupported Versions]
I created a custom ingest and it shows up in the pipeline list.  I'm getting log data from my firewall into Kibana but it looks like it's just using the default filebeat parser instead of my custom parser.  How can I direct Elasticsearch/Kibana to use my custom parser for any logs coming from the firewall?  Thanks!

**Comment by SackOfHacks** [comment]
Are you using syslog to get your firewall data into Security Onion? If so, you need to add a pipeline in the syslog ingest to direct that particular event to your custom ingest

**Reply by sturge33** [reply]
Yes, I'm sending the data via syslog to Security Onion.  Would I set that in the default syslog ingest file?

**Reply by sturge33** [reply]
From looking at the default syslog file, it appears that it will match the parser based on a value for "vendor.product".  I'm not really sure how to pass that to the syslog parser though.

**Reply by sturge33** [reply]
I think I got it.  I just needed to create a syslog parser in the custom ingest folder.  I still need to change some of the parsing and add some rules but that got me going in the right direction.  Thanks!
